src/libraries/System.Text.Encodings.Web/tools/GenUnicodeRanges/Program.cs

```diff
@@ -176,6 +176,7 @@ private static void WriteCopyrightAndHeader(StringBuilder builder)
             builder.AppendLine();
             builder.AppendLine("// This file was generated by a tool.");
             builder.AppendLine("// See src/System.Text.Encodings.Web/tools/GenUnicodeRanges");
+            builder.AppendLine("// IF YOU NEED TO UPDATE UNICODE VERSION FOLLOW THE GUIDE AT src\coreclr\System.Private.CoreLib\Tools\GenUnicodeProp\Updating-Unicode-Versions.md");
```

Comment:
Nit: The path on previous lines is using Unix style path separator. It would be nice for this path to do the same.